templates/apps/sumo-aws-apps/common-org/common-org-management.yaml

```diff
@@ -429,93 +434,7 @@ Resources:
                   StringEquals:
                     iam:AWSServiceName: 
                       - malware-protection.guardduty.amazonaws.com
-                      - guardduty.amazonaws.com
-              - Effect: Allow
```

Comment:
Why are we deleting below permissions? Aren't they used by guardduty ?

Reply:
There is some confusion here. I see it is still used for guardduty. last commit is still used for guardduty.amazonaws.com